### PR TITLE
test: add api test for exec/create apis

### PR DESCRIPTION
**Pull request by Letty5411**
Signed-off-by: letty <letty.ll@alibaba-inc.com>

**1.Describe what this PR did**
Add api test for exec/create/stop
**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

File: test/api_container_create_test.go [hunk 1]
```diff
@@ -123,3 +118,115 @@ func (suite *APIContainerCreateSuite) TestNonExistingImg(c *check.C) {
...
+	// HostConfig is nil.
+	obj := map[string]interface{}{
+		"Image":      busyboxImage,
+		"HostConfig": nil,
+	}
+
+	path := "/containers/create"
+	body := request.WithJSONBody(obj)
+
+	resp, err := request.Post(path, query, body)
```
Comment (allencloud) [inline test/api_container_create_test.go]: Could we use `resp, err := request.Post("/containers/create", query, body)` directly, like we discussed before.

If that is OK, maybe in a follow-up, we can change all of those in one commit.

File: test/api_container_start_test.go [hunk 2]
```diff
@@ -5,6 +5,7 @@ import (
 	"github.com/alibaba/pouch/test/request"
 
 	"github.com/go-check/check"
+	"net/url"
```
Comment (allencloud) [inline test/api_container_start_test.go]: Please add this package at the beginning of import.

> When importing packages, to improve readabilities, we should import package by sequence: system packages, project's own packages and third-party packages. And we should keep a blank line among these three kinds of packages;

File: test/api_container_stop_test.go [hunk 3]
```diff
@@ -5,6 +5,7 @@ import (
 	"github.com/alibaba/pouch/test/request"
 
 	"github.com/go-check/check"
+	"net/url"
```
Comment (allencloud) [inline test/api_container_stop_test.go]: The same as above.

**Comment by allencloud** [comment]
Generally LGTM, except some nits.

File: test/api_container_create_test.go (hunk 1 above)
Comment (Letty5411) [reply test/api_container_create_test.go]: ok, I'll change once I met

File: test/api_container_start_test.go (hunk 2 above)
Comment (Letty5411) [reply test/api_container_start_test.go]: OK

**Comment by allencloud** [comment]
LGTM

**Pull request merged**